common.js

```diff
@@ -103,7 +103,11 @@ function setopts (self, pattern, options) {
   if (process.platform === "win32")
     self.root = self.root.replace(/\\/g, "/")
 
-  self.cwdAbs = makeAbs(self, self.cwd)
+  // TODO: is an absolute `cwd` supposed to be resolved against `root`?
+  // e.g. { cwd: '/test', root: __dirname } === path.join(__dirname, '/test')
+  self.cwdAbs = isAbsolute(self.cwd) ? self.cwd : makeAbs(self, self.cwd)
```

Comment:
Hmm, how should I go about resolving this then?  If `makeAbs` is used, it appends an absolute path to an absolute root resulting in something like `/Users/phated/node-glob/test/Users/phated/node-glob/test/fixtures`